src/pyhub/llm/agents/tools/code_interpreter/backends/docker.py

```
@@ -0,0 +1,439 @@
+            # Check for matplotlib figures
+            for i in range(10):  # Check up to 10 figures
+                fig_path = f"/tmp/figure_{i}.png"
+                try:
+                    tar_stream = container.get_archive(fig_path)[0]
+                    with tarfile.open(fileobj=io.BytesIO(b''.join(tar_stream))) as tar:
+                        for member in tar.getmembers():
+                            f = tar.extractfile(member)
+                            if f:
+                                files[f"figure_{i}.png"] = f.read()
+                                f.close()
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)